renku/ui/cli/status.py

```diff
@@ -56,13 +56,16 @@
 
 @click.command()
 @click.pass_context
+@click.option("--ignore-deleted", is_flag=True, help="Ignore deleted paths.")
```

Comment:
Should we add a shorthand like `-i` or `-d` ?

Reply:
I've added `-i`